### PR TITLE
Add Tensorboard support

### PR DESCRIPTION
**Pull request by ericagreene**
It turns out the TensorBoard basically works out of the box with `tf.Estimator`. I read through the documentation and it turns out you do need to manually add "summary ops" to the graph for metrics you want TensorBoard to track and graph at each checkpoint. This PR adds summary ops for accuracy and AOC. I also added documentation on how to run Tensorboard locally. 

Some screenshots from a recent run with the bag of words model (it's fast to train):  

screen shot 2018-02-05 at 4 21 34 pm

screen shot 2018-02-05 at 4 21 41 pm

This closes https://github.com/conversationai/wikidetox/issues/71

### CODE REVIEW

File: conversation_classification/kaggle/model.py [hunk 1]
```diff
@@ -46,11 +52,17 @@ def estimator_spec_for_softmax_classification(logits, labels, mode):
...
+    # Holds the raw logit values
+    'logits': logits,
+
+    # Holds the class id (0,1) representing the model's prediction of the most
+    # likely species for this example.
     'classes': predicted_classes,
 
-    # Add softmax_tensor to the graph. It is used for PREDICT.
-    'probs': predicted_probs
+    # Holds the three probabilities for each prediction
```
Comment (nthain) [inline conversation_classification/kaggle/model.py]: Why are there three probabilities?

Comment (ericagreene) [reply conversation_classification/kaggle/model.py]: No idea...I think that's a copy/paste typo. I'll fix it.
